cockroachcloud/run-bulk-operations.md

```diff
@@ -108,7 +104,7 @@ Read the [`IMPORT`](../{{site.current_cloud_version}}/import.html) page for more
 
 ### Export data out of {{ site.data.products.db }}
 
-Using `EXPORT` with `userfile` is not recommended. If you need to export data from a {{ site.data.products.serverless }} cluster, you can either [set up billing for your organization](billing-management.html) to access cloud storage or export data to a local CSV file by using [`cockroach sql --execute`](../{{site.current_cloud_version}}/cockroach-sql.html#general). For example:
+Using `EXPORT` with `userfile` is not recommended. If you want to export data to a local CSV file, you can use [`cockroach sql --execute`](../{{site.current_cloud_version}}/cockroach-sql.html#general). For example:
```

Comment:
Can I access cloud storage even if I don't have billing set up for my organization?

In other words, can I use `EXPORT` to e.g. an S3 bucket now even though I did not input a credit card?

This seems to imply that the only thing I can do is export data to a local CSV file using `cockroach sql --execute`.  Is that the case?

Reply:
updated to clarify you should be able to use cloud storage here too, but i'm not sure why we only had the local example here. talking to Kathryn and will open a separate ticket to add an example here.

Reply:
SGTM, thanks!